tests/vmi_configuration_test.go

```diff
@@ -2508,15 +2508,27 @@ var _ = Describe("Configurations", func() {
 		vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
 		Expect(err).ToNot(HaveOccurred(), "Should successfully get VMI")
 
-		vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
+		vmiVirtLauncherPod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
 
 		tests.WaitAgentConnected(virtClient, vmi)
 
-		// Ensure that VMI IP Stays equal to PodIP even after Guest-Agent kicks in
+		// Ensure that VMI 'ipAddress' and 'ipAddresses' stays equal to
+		// VMI virt-launcher pod 'PodIP' and 'PodIPs' even after Guest-Agent kicks in
 		Consistently(func() bool {
 			vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred(), "Should successfully get VMI")
-			return vmi.Status.Interfaces[0].IP == vmiPod.Status.PodIP
+
+			isPodIPSynced := vmiVirtLauncherPod.Status.PodIP == vmi.Status.Interfaces[0].IP
```

Comment:
I've asked @oshoval this before; could you guys coordinate and provide a single function to make sure the IPs are consistent between the pod & the vmi ? 

pretty clear everyone will need it.

Reply:
Which package should contain it?

Reply:
Done on my PR, extracted to external file.

Reply:
This is not merged code I can't use it

Reply:
you can cherry-pick the patch , and put hold, there is just one commit that have the function that you need

Reply:
At the current state it won't work for this test for the reason I mentioned 
If this PR merge first just refactor this test
I see no reason to hold this PR

Reply:
i dont think i agree on that reason, also Miguel said it to you i think in a thread below.
would be best that you express the reason itself please that you think, anyhow do what you select,
i will continue mine.

Reply:
https://github.com/kubevirt/kubevirt/pull/4042#discussion_r478612964
I tried to answer why there isnt an issue, lets discuss it until we agree please, imo this what we should do.